rfcs/proposed/task_group_dynamic_dependencies/extended_semantics.md

```diff
@@ -546,6 +546,47 @@ is unknown, the current proposal treats it as undefined behavior.
 
 Moving this feature to `supported` requires defining the semantic for this case.
 
+### Preservation of submit arena
```

Comment:
Should this be in the potential enhancement section? Seems to me that this is a design limitation that needs to be addressed to preserve semantics. In addition to concurrency, imagine that you run a task in an arena with constraints but it executes in a different arena with the different constraints -- such as as core type or CPU mask. It might also have different FP settings.